packages/apps/fortune/exchange-oracle/server/src/common/config/database-config.service.ts

```diff
@@ -4,30 +4,70 @@ import { ConfigService } from '@nestjs/config';
 @Injectable()
 export class DatabaseConfigService {
   constructor(private configService: ConfigService) {}
+
+  /**
+   * The URL for connecting to the PostgreSQL database.
+   * Default: undefined
```

Comment:
For this cases instead of `Default: undefined`, I think is better to set `Required`